{"name": "test_finance", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_billing.TestBilling testMethod=test_finance>\n\n    @pytest.mark.order(1)\n    def test_finance(self):\n        if 1 == 1:\n>           assert False\nE           assert False\n\nFinance\\test_billing.py:16: AssertionError"}, "start": 1656752371536, "stop": 1656752376077, "uuid": "00447551-3323-4d52-a373-c22d5c7dd087", "historyId": "78357a1b80dc9ff97f255959f02d7457", "testCaseId": "93f38f15350138dfa98674a11da4b44f", "fullName": "Finance.test_billing.TestBilling#test_finance", "labels": [{"name": "tag", "value": "order(1)"}, {"name": "parentSuite", "value": "Finance"}, {"name": "suite", "value": "test_billing"}, {"name": "subSuite", "value": "TestBilling"}, {"name": "host", "value": "DESKTOP-DG2A6BK"}, {"name": "thread", "value": "15884-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Finance.test_billing"}]}